tailwind.config.js

```diff
@@ -1,229 +1,5 @@
-        wiggle: "url('/_assets/img/landing/wiggle.svg')",
-        orangeBow: "url('/_assets/img/landing/about-container.svg')",
-        blueWave: "url('/_assets/img/landing/footer-container.svg')",
-        topWaveFromWhite: "url('/_assets/img/landing/top-wave.svg')",
-      },
-      backgroundSize: {
-        '100%': '100% 100%',
-      },
-      minHeight: {
-        8: '32px',
-        '1/2': '50vh',
-        '1/4': '25vh',
-      },
-      maxWidth: {
-        xs: '300px',
-        65: '260px',
-        '30p': '30%',
-      },
-      fontSize: {
-        'base-plus': ['1.1875rem', '1.22'],
-        '1.5xl': ['1.3125rem', '1.22'],
-        '2.5xl': ['1.7rem', '1.22'],
-        '3.5xl': ['2rem', '1.22'],
-      },
-      width: {
-        '1/8v': '12.5vw',
-        '1/4v': '25vw',
-        '1/3v': '33vw',
-      },
-      height: {
-        630: '630px',
-      },
-      cursor: {
-        'zoom-in': 'zoom-in',
-      },
-    },
-    screens: {
-      mobile: '500px',
-      mobileExt: '550px',
-      sm: '800px',
-      md: '1024px',
-      lg: '1216px',
-      xl: '1300px',
-      print: { raw: 'print' },
-    },
-  },
-  plugins: [
-    require('tailwindcss-animate'),
-    plugin(function ({ addUtilities, addComponents }) {
-      // Custom utilities all start with `special-*`
-      // They use css that is not covered by tailwind
-      addUtilities({
-        '.special-border-half-transparent': {
-          borderTopColor: 'transparent',
-          borderLeftColor: 'transparent',
-        },
-        '.special-reset-list-counter': {
-          'counter-reset': 'list-counter',
-        },
-        '.special-content-list-counter': {
-          content: 'counter(list-counter)',
-        },
-        '.special-increment-list-counter': {
-          'counter-increment': 'list-counter',
-        },
-        '.special-content-space': {
-          content: '" "',
-        },
-        '.special-content-gt': {
-          content: '">"',
-        },
-        '.special-hyphens-auto': {
-          hyphens: 'auto',
-        },
-        '.special-hyphens-initial': {
-          hyphens: 'initial',
-        },
-        'special-no-page-breaks-inside': {
-          'page-break-inside': 'avoid',
-        },
-      })
-
-      // add classes of serlo-components to autocomplete
-      addComponents(extractCSSClasses())
-    }),
-  ],
-}
-
-function extractCSSClasses() {
-  try {
-    const css = require('fs').readFileSync(
-      require('path').join(
-        __dirname,
-        '/src/assets-webkit/styles/serlo-tailwind.css'
-      ),
-      'utf-8'
-    )
-
-    const regex = /\.serlo\-[^ \:\{\n,]+/gm
-    let m = ''
-    const components = {}
-
-    while ((m = regex.exec(css)) !== null) {
-      // The result can be accessed through the `m`-variable.
-      components[m[0]] = {}
-    }
-
-    return components
-  } catch (error) {
-    // don't run on client, no problem
-  }
+  content: ['./src/**/*.{js,ts,jsx,tsx}', './shared/tailwind-base.cjs'],
```

Comment:
@elbotho Do you remember why we included the tailwind config into the config path? I thought it was about some editor colors - but looking through the config this looks obsolete.

I think we can remove it.

Reply:
digging through file history, I found out that the first versions looked like this: https://github.com/serlo/frontend/blob/81572b71c0f39127de61d59ed010b8b0ca5bd3dd/tailwind.config.js

I moved this out of the file, and now it's not necessary anymore - prefer `serlo-tailwind.css`.

Reply:
Thanks for checking!